app/controllers/application_controller.rb

```diff
@@ -1,3 +1,18 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+
+
+  private
+
+    def get_current_office
+      if params[:office_id].present?
+        @current_office = Office.find(params[:office_id])
+      else
+        @current_office = Office.first
+      end
```

Comment:
We should use a more verbose but cleaner version like this:

```
def current_office
  @current_office ||= begin
    if params[:office_id].present?
      Office.find(params[:office_id])
    else
      Office.first
    end
  end
end
```

Then remove all references to `get_current_office` and replace most instances of `@current_office` with `current_office`. Only use `@current_office` when it is followed by `= ...`.